bin/ch-grow.py.in

```diff
@@ -160,6 +161,15 @@ defined of --storage, $CH_GROW_STORAGE, and %s.
    if (cli.parse_only):
       sys.exit(0)
 
+   # Line noise in the second option is for the git@github.com:... variant.
+   # Username and host characters from https://tools.ietf.org/html/rfc3986.
+   if (re.search(r"""  ^((git|git+ssh|http|https|ssh)://
```

Comment:
Can you provide a one line comment that gives a bit more detail on what this actually does?